public/css/pivot.css

```diff
@@ -1,13 +1,13 @@
 html {
```

Comment:
It is generally considered bad practice to check in commented-out code. Is there a reason that we should make an exception in this case?